blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/Instrumentation.java

```diff
@@ -173,4 +168,18 @@ public static void markMetricsWithShortDelayReceived() {
     public static void markMetricsWithLongDelayReceived() {
         metricsWithLongDelayReceived.mark();
     }
+
+    public static Meter getIngestedMetricsMeter(String tenantId) {
+        if ( StringUtil.isNullOrEmpty(tenantId) ) {
+            return null;
+        }
+        return Metrics.meter(Instrumentation.class, "tenants", tenantId, "Data Points Ingested");
```

Comment:
Yes. Internally, in Codahale/Dropwizard, the meters etc is kept in a MetricsRegistry class, which holds all these objects in a HashMap.